dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/JSONFieldsTest.java

```diff
@@ -309,6 +310,19 @@ public void testRenameAndOmitExtFields() throws Exception {
         getHttpServlet("/extFields/CreateLogs", server_xml); //see omitted field
         assertNull("The extension field name was not omitted", server_xml.waitForStringInLogUsingMark("ext_testExtension"));
```

Comment:
should also check to make sure MY_EXTENSION doesn't appear in the log here (in case the reconfiguration doesn't happen properly)